examples/user_guide/14-Data_Pipelines.ipynb

```diff
@@ -138,12 +141,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def rolled_data(symbol, rolling_window, **kwargs):\n",
-    "    curve = load_symbol(symbol)\n",
-    "    return rolling(curve, rolling_window=rolling_window)\n",
-    "    \n",
-    "rolled_dmap = hv.DynamicMap(rolled_data, kdims='Symbol',\n",
-    "                            streams=[stream]).redim.values(Symbol=stock_symbols)\n",
+    "rolled_dmap = dmap.apply(rolling, rolling_window=slider.param.value)\n",
     "\n",
     "rolled_dmap"
```

Comment:
I would consider saying something about what calls to `.apply` returns - another DynamicMap that nests the original within the supplied function.